ducktape/decorate.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2015 Confluent Inc.
```

Comment:
I guess we're following py.test's lead here :) Not really sure "mark" addresses my initial concern, which was that every decorator you could imagine would end up in here, e.g. @ignore, @timeout, etc. as you mentioned below. Maybe there aren't actually that many though.